tests/test_pyscalcli.py

```diff
@@ -273,6 +271,39 @@ def test_pyscalcli_stdout_output(capsys, mocker):
     assert "SOF3" in captured.out
 
 
+def test_pyscalcli_let_pd_and_imb(capsys, mocker):
+    """Test that we can write to stdout"""
+    scalrec_file = (
+        Path(__file__).absolute().parent / "data/pyscal_config_simple_let_hyst.xlsx"
```

Comment:
Can you convert the xlsx to csv instead, then it will be viewable in github and better for version control

Reply:
done.